Misc/NEWS.d/next/Build/2018-07-03-18-36-35.bpo-34032.bSAQzz.rst

```diff
@@ -0,0 +1,3 @@
+Build system has been modified to allow /usr/lib64 as well as /usr/lib into
+install path as well as to the search path for the Linux distributions which
+distinguish between 32bit and 64bit libraries.
```

Comment:
Can you mention the new --with-custom-platlibdir=<libdirname> configure option somewhere?

Nitpick: I would write "32 bit and 64 bit libraries" ;-) (or "32-bit and 64-bit libraries")